### PR TITLE
Add section to README highlighting key use-cases for the API

### PR DESCRIPTION
**Pull request by nickebbitt**
# Checklist

- [ ] You changed or added to the documentation

### CODE REVIEW

**Comment by nickebbitt** [comment]
Would I be right in saying our Docker pipelines also use the API?

File: README.md [hunk 1]
```diff
@@ -14,7 +14,17 @@ The AdoptOpenJDK API provides a way to consume JSON information about the AdoptO
...
 
 For more information, including example queries, please look at [STRUCTURE.md](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md)
+
+## Who's using the AdoptOpenJDK API?
+
+There are various consumers of the AdoptOpenJDK API, from individuals to organisations. 
+
+The following list highlights the key consumers and their use-cases:
+
+- <https://adoptopenjdk.net/> - the API drives the release listings on the AdoptOpenJDK website allowing individuals to download the JDK distribution of their choice
```
Comment (karianna) [inline README.md:28]: Make a proper markdown link?

File: README.md [hunk 2]
```diff
@@ -14,7 +14,17 @@ The AdoptOpenJDK API provides a way to consume JSON information about the AdoptO
...
+The API is documented via swagger.  The swagger documentation can be viewed at: [swagger-ui](https://api.adoptopenjdk.net/swagger-ui). 
 The open api definition for this can be viewed at [openapi](https://api.adoptopenjdk.net/openapi).
 
 For more information, including example queries, please look at [STRUCTURE.md](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md)
+
+## Who's using the AdoptOpenJDK API?
+
+There are various consumers of the AdoptOpenJDK API, from individuals to organisations. 
+
+The following list highlights the key consumers and their use-cases:
```
Comment (karianna) [inline README.md:26]: Probably need to say this is a small subset, and perhaps provide a link to our dash.adoptopenjdk.net showing the 200M+ downloads that have been served via hte API

**Comment by nickebbitt** [comment]
@karianna I've added the link to the stats dashboard as requested, whilst doing so I found the dashboard to be a little temperamental with numbers not showing and what appears to be dependencies on the v2 API also

Mentioning it here but should I raise an issue on its repo (if that's the right one)? https://github.com/AdoptOpenJDK/openjdk-dashboard-v2

Screen Shot 2020-11-12 at 9 56 18 PM

**Comment by nickebbitt** [comment]
actually, maybe that's just the wrong dashboard I'm linking to, should it be the v2 URL?

https://dash-v2.adoptopenjdk.net/

**Comment by karianna** [comment]
> @karianna I've added the link to the stats dashboard as requested, whilst doing so I found the dashboard to be a little temperamental with numbers not showing and what appears to be dependencies on the v2 API also, with 404s against one of the URLs it's hitting.
> 
> Mentioning it here but should I raise an issue on its repo (if that's the right one)? https://github.com/AdoptOpenJDK/openjdk-dashboard-v2
> 
> Screen Shot 2020-11-12 at 9 56 18 PM

yes please - we should either fix that or redirect to the new dashboard

**Comment by nickebbitt** [comment]
@karianna hey, I've fixed the dashboard link. Also added the Docker image creation as a use-case.

This one is possibly good to merge?

**Pull request merged**